duijf-julia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:edge
+MAINTAINER robin@kingsquare.nl
+
+RUN apk add --no-cache julia --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ --allow-untrusted
+
+# Create working dir
+RUN mkdir -p /var/app
+COPY ./raffler.jl /var/app
```

Comment:
thats my guess... hence my echo-ing of the docker version in travis... (they are running 1.8 currently)